### ISSUE TITLE
TeachingBubble headline is missing header level and cannot be customized

### ISSUE THREAD

**Issue by shensi2727** [microsoft/fluentui#15979]
https://github.com/microsoft/fluentui/blob/c6ae2b350037deb4e0bea07cdbdfe2e93455f4b1/packages/react-internal/src/components/TeachingBubble/TeachingBubbleContent.base.tsx#L99

This headline tag switching will never work as 'div' since the type of headline property is set as string | undefined. There is no other case allowed customized element passing into headline property and render a specific element instead of a "p" tag with string.

There is an accessibility bug I looked into recently that TeachingBubble headline needs a "h2" tag for accessibility heading level. Thus I found it hard to customize headline in TeachingBubble component and not able to give it an "h2" tag. Also, if use "aria-" as a workaround, there is only "aria-role" indicating a "heading" role, but still lacks "aria-level" to indicate heading levels.

**Comment by paulgildea** [comment]
@shensi2727 Thanks for filing this issue. We'll take a look. Since you were able to diagnose the issue are you interested in contributing a fix?

**Comment by mashenoy** [comment]
@behowell - Don't see any movement with this issue since Nov 2020. Did you get a chance to look at this issue ?
